### DISCUSSION TITLE
Episode 172: Sean Parent on Flash, Chains & Memory Safety

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#64] [Episode]
Episode Link: https://adspthepodcast.com/2024/03/08/Episode-172.html

Feel free to write comments or ask questions 🙂

**Comment by BraxtonMDevelops** [comment]
Glad to listen to a longer episode of ADSP

**Comment by kentf** [comment]
The beginning of the podcast was more akin to the found footage genre, like Blair Witch or The Troll Hunter. 😄 

Next up; _ADSP in New York city. Will they find the correct Adobe office in time? Is there any hope of interviewing Sean Parent before the time runs out? May the answer be in episode 172 of ADSP - Algorithms, Datastructures, and Stray Podcasters_.

![bryce_and_conor_lost_in_new_york](https://github.com/codereport/adsp2/assets/5132364/816c7021-fc8a-4b09-8ca1-8f2195f37178)

**Comment by Brian-M-J** [comment]
There was a lot of discussion on the C++ implementation of senders & receivers. Sean Parent also worked on Hylo, so I wanted to know his thoughts on its implementation there. What are the tradeoffs that Hylo makes?

**Reply by Brian-M-J** [reply]
Also, the talk mentioned around 36:36 isn't linked in the show notes.

**Reply by codereport** [reply]
This is the talk: https://youtu.be/5Zt5TNqKhcA?si=4Ul4pFsQAwd9Ut1u&t=1875 Will add it to the show notes! Thanks 🙏
